earth_enterprise/src/portableserver/build.py

```diff
@@ -140,7 +140,8 @@ def get_version(self):
         """Parses version information, and sets member variables."""
 
         if self.base_version is None:
-            path = os.path.join(self.source_dir, 'fusion_version.txt')
+            # This was fusion_version.txt; file has moved.
```

Comment:
I don't think comments about non-existent files are very helpful to future developers.  I think we can do without this one.